### PR TITLE
feat: add contract URI to be opensea compatible

### PR DESCRIPTION
**Pull request by willKim19**
Based on this post: https://docs.opensea.io/docs/contract-level-metadata

### CODE REVIEW

File: contracts/MemberToken.sol [hunk 1]
```diff
@@ -28,6 +28,12 @@ contract MemberToken is ERC1155Supply, Ownable {
         controllerRegistry = IControllerRegistry(_controllerRegistry);
     }
 
+    // Provides metadata value for the opensea wallet. Must be set at construct time
+    // Source: https://www.reddit.com/r/ethdev/comments/q4j5bf/contracturi_not_reflected_in_opensea/
+    function contractURI() public view returns (string memory) {
+        return "https://orcaprotocol-nft.vercel.app/assets/contract-metadata";
```
Comment (jtlacci) [inline contracts/MemberToken.sol:34]: we need to make this updateable

**Review by jtlacci — CHANGES_REQUESTED** [review]
we shou

File: contracts/MemberToken.sol (hunk 1 above)
Comment (jtlacci) [reply contracts/MemberToken.sol:34]: that could always change in the future though we don't want to be locked in to a single server

Comment (willKim19) [reply contracts/MemberToken.sol:34]: fair enough

**Review by jtlacci — APPROVED** [review]

**Pull request merged**